packages/synapse-react-client/src/components/GenericCard/Linkify.test.tsx

```diff
@@ -121,6 +121,18 @@ describe('Linkify tests', () => {
     )
   })
 
+  test('auto-links doi with Synapse ID', () => {
```

Comment:
Adding a new test, but there are existing tests for Synapse IDs.  See 'auto-link only the Synapse ID' and 'auto-links a Synapse ID'